wiki/Help_Centre/Restricted_Mode/en.md

```diff
@@ -0,0 +1,58 @@
+- **Make sure you do respect the rules.** Restrictions are punishments for breaking the rules. If you continue doing so while restricted, the cooldown before you can appeal will be reset. This includes continuing to cheat, or creating or accessing new accounts to evade the restriction.
+- **Follow the support team's instructions.** The three-month cooldown imposed on accounts is a chance to take time off and reflect on your actions. If you attempt appealing before that time, or continue trying to contact the support team to shorten the cooldown, you will be ignored.  If you are unsure when your cooldown expires, when you will be allowed to appeal, or which of your accounts you should be playing on and appealing for, you are welcome to send us an email to ask. Just make sure not to send any appeals before the date given to you!
+- **Do not hesitate to contact us if you have any questions.** As long as you are not trying to circumvent punishment or shorten the cooldown, we will happily answer any questions you have about your account's restriction!
+
+## Three months have passed. How do I appeal?
+
+You can appeal by sending an email to [accounts@ppy.sh](mailto:accounts@ppy.sh). Again, we will need your username, and make sure that you are sending it from the email address attached to your account! After you send your appeal, expect a reply within one week. If you have not received one then, please contact us on twitter or by email - there is a chance we have not received it or closed it by accident!
+
+When trying to appeal, you are free to write what you want in your message! We do however have expectations for what the appeal should include. Appeals do not have to be full-length essays - as long as you are honest and genuine, your appeal will be sufficient. We are looking to make sure you know what you were restricted for and you are aware of what rules you have broken. There is no point in unrestricting a user who will continue breaking the rules after being unrestricted.
+
+We expect the following points in appeals:
+- **Account history.** It does not need to be too detailed. This should include all your breaches of the major rules (multi-accounting, account sharing, cheating, or major silences). You do not need to remember every date, every silence, or every account you created, as long as you are honest with what you did. This includes any rules you have broken while restricted if you did.
+- **Why you broke the rules.** An explanation as to what led you to decide to create new accounts, give your account away, cheat, or others, will help us (and you) determine whether or not you are at risk of offending again. If you continued offending while restricted, we will also be interested in knowing why.
```

Comment:
`- **Why you broke the rules.** An explanation as to what led you to decide to create new accounts, give your account away, cheat, etc. will help us (and you) determine whether or not you are at risk of offending again. If you continued offending while restricted, we are also interested in knowing why.`